src/Microsoft.AspNetCore.SignalR.Core/Internal/DefaultHubDispatcher.cs

```diff
@@ -209,6 +209,11 @@ public override IReadOnlyList<Type> GetParameterTypes(string methodName)
             }
         }
 
+        private string BuildUnexpectedErrorMessage(string methodName, Exception exception)
+        {
+            return $"An unexpected error occurred invoking '{methodName}' on the server. {exception.GetType().Name}: {exception.Message}";
```

Comment:
null check on exception?

Reply:
It can't really be `null`. It's passed a value from a `catch` block. At most it would be an `Assert` and we don't really do those kind of asserts.

Reply:
Wasn't 100% positive of that so I verified :): https://sharplab.io/#v2:C4LgTgrgdgPgAgBgARwIwG4CwAoOBmFAJiQGEkBvHJalAtANhQBYVUAOACgEoBIS7GoKTAwATwpUh1HsAAWYAPYB3JFAgAbdVgFSkAXyQBjAIbBDsjgFEAHoYCmAB2ABLBVCR3rXCTt000AJwcAESeSM4Azqoa6gD8SMFIANRIHGEAvOnRmlxc2n56ktSF2HpAA=

Throwing `null` appears to yield a `NullReferenceException`, so the exception value wouldn't be null.

Reply:
You can throw non-Exceptions via IL but we're explicitly catching `Exception`